src/_includes/layouts/event.html

```diff
@@ -37,8 +37,25 @@ <h1>{{ title }}</h1>
 
 				{% if calendarLinks.isPastEvent %}
 				<div>{% include partials/calendarLinks.html %}{% include partials/join-discord-button.html %}</div>
-				{% endif %} {{ content }}
+				{% endif %} {{ content }} {% if speakers.size > 1 %}
+				<h3>Speakers</h3>
+				{% elsif speakers.size == 1 %}
+				<h3>Speaker</h3>
+				{% else %} {% endif %}
```

Comment:
I love this attention to detail — and I love that we won't ever have to mind this manually ourselves! 🥳